tests/idea-space.spec.ts

```diff
@@ -63,13 +65,13 @@ test.describe('comment migration', () => {
 
     test("should update comment author", async () => {
         const strapi = await strapiConnect();
-        const params = { populate: ["user"] };
-        await strapi.db.query("api::comment.comment").update({ where: { author: user.username }, data: { user: null } });
+        const params = { where: { id: commentId }, populate: ["user"] };
+        await strapi.db.query("api::comment.comment").update({ where: { id: commentId }, data: { user: null } });
         let comment = await strapi.db.query("api::comment.comment").findOne(params);
```

Comment:
Hey @becevka ,
Why in this case the expected is null ?

Reply:
on line 69 we updated comment and set user field to be Null. Here we verify user is null, run migration and then verify that user was found and comment was updated.